### PR TITLE
fixing mostly typos and spelling mistakes

### PR DESCRIPTION
**Pull request by mbergumi**
They clutter results when running code inspections

### CODE REVIEW

File: generators/client/templates/src/main/webapp/app/admin/gateway/_gateway.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h3 data-translate="gateway.routes.title">Current routes</h3>
                 <div ng-show="{{route.serviceInstances.length == 0}}" class="label label-danger" data-translate="gateway.routes.error">
                     Warning: no server available!
                 </div>
-                <table class="tabletable-responsive">
+                <table class="table-responsive">
```
Comment (ruddell) [inline generators/client/templates/src/main/webapp/app/admin/gateway/_gateway.html]: This should be `table table-responsive`.  Everything else looks 👍

**Comment by jdubois** [comment]
Great, thanks!! I'll merge this soon.

**Comment by mbergumi** [comment]
@ruddell fixed! 
@jdubois you're welcome. Thanks to you too, Jhipster has been an amazing support for self-improvement with Spring Framework.

File: generators/client/templates/src/main/webapp/app/admin/gateway/_gateway.html (hunk 1 above)
Comment (mbergumi) [inline generators/client/templates/src/main/webapp/app/admin/gateway/_gateway.html]: done

**Pull request merged**